pt_br.lang

```diff
@@ -1,255 +1,388 @@
+	"gui.xaero_archery_status" : "Exibir flechas rest.",
+	"gui.xaero_sprint" : "Better Sprint",
+	"gui.xaero_sneak" : "ToggleSneak",
+	"gui.xaero_enchants_color" : "Cores de encant.",
+	"gui.xaero_durability" : "Duração",
+	"gui.xaero_hunger_setting" : "Notifi. de fome",
+	"gui.xaero_hp_setting" : "Notifi. de saúde",
+	"gui.xaero_explosion_setting" : "Notifi. de explosões",
+	"gui.xaero_being_shot_setting" : "Notifi. ao ser ferido",
+	"gui.xaero_xp_setting" : "Quantidade de XP",
+	"gui.xaero_custom_settings" : "Personalizar config.",
+	"gui.xaero_reset_defaults" : "Restaurar config.",
+	"gui.xaero_quick_use" : "Acesso rápido de item",
+	"gui.xaero_show_enchants" : "Exibir encant.",
+	"gui.xaero_cave_maps" : "Mapas de cavernas",
+	"gui.xaero_display_teams" : "Exibir outras equipes",
+	"gui.xaero_display_waypoints" : "Waypoints no mapa",
+	"gui.xaero_display_players" : "Exibir jogadores",
+	"gui.xaero_display_mobs" : "Exibir criaturas",
+	"gui.xaero_display_items" : "Exibir itens",
+	"gui.xaero_display_other" : "Exibir outros",
+	"gui.xaero_zoom" : "Zoom",
+	"gui.xaero_minimap_size" : "Tamanho do minimapa",
+	"gui.xaero_entity_amount" : "Quant. de entidades",
+	"gui.xaero_black" : "Preta",
+	"gui.xaero_dark_blue" : "Azul-escuro",
+	"gui.xaero_dark_green" : "Verde-escuro",
+	"gui.xaero_dark_aqua" : "Aqua-escuro",
+	"gui.xaero_dark_red" : "Vermelho-escuro",
+	"gui.xaero_dark_purple" : "Roxo-escuro",
+	"gui.xaero_gold" : "Dourada",
+	"gui.xaero_gray" : "Cinza",
+	"gui.xaero_dark_gray" : "Cinza-escuro",
+	"gui.xaero_blue" : "Azul",
+	"gui.xaero_green" : "Verde",
+	"gui.xaero_aqua" : "Aqua",
+	"gui.xaero_red" : "Vermelha",
+	"gui.xaero_purple" : "Roxa",
+	"gui.xaero_yellow" : "Amarela",
+	"gui.xaero_white" : "Branca",
+	"gui.xaero_percent" : "Porcentagem",
+	"gui.xaero_numeral" : "Números",
+	"gui.xaero_both" : "Ambos",
+	"gui.xaero_tiny" : "Mínimo",
+	"gui.xaero_small" : "Pequeno",
+	"gui.xaero_medium" : "Médio",
+	"gui.xaero_large" : "Grande",
+	"gui.xaero_pvp_sprint" : "PVP Sprint",
+	"gui.xaero_quick_eat" : "Quick Eat",
+	"gui.xaero_toggle_sneak" : "ToggleSneak",
+	"gui.xaero_quick_use_1" : "Acesso rápido 1",
+	"gui.xaero_quick_use_2" : "Acesso rápido 2",
+	"gui.xaero_quick_use_3" : "Acesso rápido 3",
+	"gui.xaero_quick_use_4" : "Acesso rápido 4",
+	"gui.xaero_quick_use_5" : "Acesso rápido 5",
+	"gui.xaero_quick_use_6" : "Acesso rápido 6",
+	"gui.xaero_quick_use_7" : "Acesso rápido 7",
+	"gui.xaero_quick_use_8" : "Acesso rápido 8",
+	"gui.xaero_quick_use_9" : "Acesso rápido 9",
+	"gui.xaero_zoom_in" : "Aumentar zoom",
+	"gui.xaero_zoom_out" : "Reduzir zoom",
+	"gui.xaero_on" : "Sim",
+	"gui.xaero_off" : "Não",
+	"gui.xaero_restores" : "Restaura",
+	"gui.xaero_preset_topleft" : "Superior esquerdo",
+	"gui.xaero_preset_topright" : "Superior direito",
+	"gui.xaero_waypoint_teleport" : "Teletransportar",
+	"gui.xaero_waypoints_key" : "Waypoints",
+	"gui.xaero_display_coords" : "Exibir coordenadas",
+	"gui.xaero_disable_enable" : "Ativar/Desativar",
```

Comment:
The order here shouldn't be changed and should match the following pattern: `<Text_disable_here>`/`<Text_enable_here>`.

It's also a quick reminder for any future reviewer (most likely for me).